src/components/input-field.jsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const InputField = ({ captureField, name }) => <label> {name.replace(name.charAt(0), name.charAt(0).toUpperCase())}: <input type='text' name={name} id={name} onChange={(e) => captureField(e)}/>  <br/></label>;
```

Comment:
name.charAt(0) is a very vanilla JS (long hand way) way of writing name[0]. I'd be curious why you chose charAt() instead of position identification with [var][position]

Reply:
I just googled and the first solution was to use charAt :)